optuna/integration/_lightgbm_tuner/__init__.py

```diff
@@ -16,8 +16,9 @@ def train(*args: Any, **kwargs: Any) -> Any:
 
     It tunes important hyperparameters (e.g., ``min_child_samples`` and ``feature_fraction``) in a
     stepwise manner. It is a drop-in replacement for `lightgbm.train()`_. See
-    `a simple example of LightGBM Tuner <https://github.com/optuna/optuna/blob/master/examples/lig
-    htgbm_tuner_simple.py>`_ which optimizes the validation log loss of cancer detection.
+    `a simple example of LightGBM Tuner <https://github.com/optuna/optuna/blob/master/examples/
+    ligthgbm/lig htgbm_tuner_simple.py>`_ which optimizes the validation log loss of cancer
```

Comment:
Could you fix the path?
```suggestion
    lightgbm/lightgbm_tuner_simple.py>`_ which optimizes the validation log loss of cancer
```